daml-assistant/daml-helper/src/DA/Daml/Helper/Ledger.hs

```diff
@@ -644,3 +648,25 @@ sanitizeToken :: String -> String
 sanitizeToken tok
   | "Bearer " `isPrefixOf` tok = tok
   | otherwise = "Bearer " <> tok
+
+-- | Report on Ledger Use.
+runLedgerMeteringReport :: LedgerFlags -> (Maybe String) -> (Maybe String) -> (Maybe String) -> IO ()
```

Comment:
let’s move the parsing to the CLI parser instead of passing strings here

Reply:
Done.